### DISCUSSION TITLE
Change CBC parameters

### DISCUSSION THREAD

**Discussion by fuejak01** [google/or-tools#2645] [Linear Solver questions] [answered]
Hi,

I'm using the CBC solver with ortools for an 0-1 MIP. (C++)
Is there a way to modify some details in the algorithm. For example, I want to change the way the next instance is picked in the branching tree and I want to change the number of steps between two cuts.

Thanks for you help,
Jakob

**Comment by Mizux** [comment, marked as answer]
https://github.com/google/or-tools/blob/b37d9c786b69128f3505f15beca09e89bf078a89/ortools/linear_solver/cbc_interface.cc#L483-L492

**Reply by fuejak01** [reply]
Thanks for the answer!

I'm not very experienced with C++, could you give me an example how this function is used.
